Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 properties([
   disableConcurrentBuilds(abortPrevious: true),
   buildDiscarder(logRotator(numToKeepStr: '7')),
-  pipelineTriggers([cron('7 18 * * 5')])
+  pipelineTriggers([cron('7 8 * * 5')])
```

Comment:
could have been 8am UTC, but I like the 8:07am UTC ;)

Reply:
Yes, because 8:00 AM is way too early for me. 😉